### PR TITLE
feat(app-store-connect): APIs to get bundle ID and profile relationships

### PR DESCRIPTION
**Pull request by lucasfernog-crabnebula**
This changes the app-store-connect package to include APIs to:
- list the capabilities associated with a bundle ID
- list the profiles associated with a bundle ID
- enable a capability for a bundle ID
- get the bundle ID associated with a profile
- list the certificates associated with a profile

### CODE REVIEW

File: app-store-connect/src/profile_api.rs [hunk 1]
```diff
@@ -191,7 +213,7 @@ pub struct ProfileAttributes {
     pub platform: String,
     pub profile_content: String,
     pub uuid: String,
-    pub created_date: String,
+    pub created_date: Option<String>,
```
Comment (indygreg) [inline app-store-connect/src/profile_api.rs:216]: I guess you encountered a missing / null value in the wild and that's why this change was made?

**Review by indygreg — APPROVED** [review]
This seems very reasonable to me. Thank you for the contribution!

File: app-store-connect/src/profile_api.rs (hunk 1 above)
Comment (lucasfernog-crabnebula) [reply app-store-connect/src/profile_api.rs:216]: that's correct

**Comment by indygreg** [comment]
Ok. Rebased locally and pushed to the `main` branch. I hope to ship a new release with these features in the next few hours.
